packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts

```diff
@@ -51,7 +51,7 @@ export interface PassProps {
      *
      * @default No injected result
      */
-    readonly result?: any;
+    readonly result?: {[key: string]: any};
```

Comment:
Call this out as a BREAKING CHANGE in the commit message that you merge.

Reply:
will do